bitnami/kafka/templates/statefulset.yaml

```diff
@@ -210,7 +210,7 @@ spec:
             {{- else }}
             - name: KAFKA_CFG_ADVERTISED_LISTENERS
               {{- if .Values.advertisedListeners }}
-              value: {{ .Values.advertisedListeners }}
+              value: {{ include "kafka.advertisedListeners" . }}
```

Comment:
You'll also have to adapt the line below in the **scripts-configmap.yaml**:

- https://github.com/bitnami/charts/blob/master/bitnami/kafka/templates/scripts-configmap.yaml#L110

Reply:
Same concern about avoiding new macros.